db/migrate/20221104141836_create_rejection_feedbacks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRejectionFeedbacks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :rejection_feedbacks do |t|
+      t.boolean :helpful
```

Comment:
Should this not allow nulls?

```suggestion
      t.boolean :helpful, null: false
```

Reply:
I was thinking set it to `true` if answered yes, `false` if no otherwise leave it as `nil`?

Reply:
If it's `nil` could there just be no row at all? ie we only add a row when they leave the feedback...?